eea/versions/versions.py

```diff
@@ -138,7 +139,14 @@ def versions(self):
             brains = cat.unrestrictedSearchResults(**query)
         else:
             brains = cat(**query)
-        objects = [b.getObject() for b in brains]
+        objects = []
+        for b in brains:
+            try:
+                objects.append(b.getObject())
+            except Unauthorized:
+                # because of the unrestricted search done above, this
+                # might happen, and we don't need a crash
+                pass
```

Comment:
@valipod I would use `continue` instead of `pass` here and also a `logger.warn` would be nice to have.

```Python
except Unauthorized as err:
    logger.warn(err)
    continue
```

Reply:
done, see bellow